docs/user-guide/security-features.md

```diff
@@ -1,152 +1,152 @@
 
-## Security
+## Configuring Security Policies
 
-You can check Vulnerabilities for all your applications in one place. On the Home page, there is an option named `Security`. Here, you can see a list of applications under the `Security Scan` tab. Here all the applications are listed which have the `Scan for Vulnerabilities` feature enabled. You can see the vulnerability count along with the Severity Level for all your applications.
+You can establish security policies for their vulnerabilities through the Security Policies tab, which can be accessed from the left pane by navigating to Security and selecting Security Policies. Policies are implemented in a hierarchical order, following a specific sequence. The order of implementation is as follows, starting from the highest level:
 
-**Note:-**
+* **Global**
+* **Cluster**
+* **Environment**
+* **Application**
 
-It displays the “Vulnerability count and Severity Level” on a priority basis. And critical level has the highest priority, so it displays the critical level vulnerabilities and there counts if any application is having critical vulnerability in it.
+Policies are implemented in a hierarchical order, with the following sequence: Global, Cluster, Environment, and Application. Higher-level policies take precedence over lower-level policies, ensuring a systematic and structured enforcement of security measures.
 
-You can directly `Search` your application using the Search bar or you can `filter out` your requirement according to Severity, Clusters, and Environment.
-
-![](../.gitbook/assets/security-feature-securitytab-security-scan%20%282%29.png)
-
-Now if you click on the severity level of vulnerability it will show you the list of all vulnerabilities along with other details.
-
-![](../.gitbook/assets/security-feature-securitytab-list-vulnerability%20%282%29.png)
-
-## Security Policies:
+![](../.gitbook/assets/security-feature-global-security-policies.png)
 
-Users can define Security policies for their vulnerabilities under `Security Policies` Tab.
+**Some examples of how policies can be defined**
 
-Home Page-&gt; Security - &gt; Security Policies
+Users can block all the critical vulnerabilities and allow the moderate and low vulnerabilities or Users can block all vulnerabilities or users can block all vulnerabilities for one application and can block only Critical vulnerabilities for other applications.
 
-Policies can be defined to different levels-
+## Configure Global Security Policy
 
-* Global
-* Cluster
-* Environment
-* Application
+Within the Global Security Policies, there are two options available: Block and Allow.
+If critical severity levels are blocked in the Global Security Policy, the same blocking will be applied to the Cluster Security Policy. Similarly, if the global policy is modified to allow critical levels, it will also allow them in Cluster Security Policies. 
```

Comment:
If critical severity levels are blocked in the `Global Security Policy`, the same blocking will be applied to the `Cluster Security Policy`. Similarly, if the global policy is modified to allow critical levels, it will also allow them in `Cluster Security Policies`.